src/XRP/model/xrp-account-set.ts

```diff
@@ -0,0 +1,59 @@
+import { AccountSet } from '../Generated/web/org/xrpl/rpc/v1/transaction_pb'
+
+/*
+ * Represents an AccountSet transaction on the XRP Ledger.
+ * An AccountSet transaction modifies the properties of an account in the XRP Ledger.
+ * @see: https://xrpl.org/accountset.html
+ */
+export default class XRPAccountSet {
+  /**
+   * Constructs an XRPAccountSet from an AccountSet.
+   *
+   * @param accountSet an AccountSet (protobuf object) whose field values will be used
+   *                to construct an XRPAccountSet
+   * @return an XRPAccountSet with its fields set via the analogous protobuf fields.
+   * @see https://github.com/ripple/rippled/blob/3d86b49dae8173344b39deb75e53170a9b6c5284/src/ripple/proto/org/xrpl/rpc/v1/transaction.proto#L100
```

Comment:
are we doing `-` between doc @params etc?

Reply:
we haven't done in any of our other `XRPWhatever` model objects... so for now I'm going to leave it like this.  When the ESLint configurations that Hans has been testing out on `xpring-common-js`, then I suspect we'll do a massive linting overhaul all at once.